### ISSUE TITLE
Missing content on engadget.com

### ISSUE THREAD

**Issue by PedroAmorim** [mozilla/readability#437]
Hi!

Most articles on engadget.com have missing content on reader view, the first paragraph is always lost.
- [article 1](https://www.engadget.com/2018/04/18/buick-enspire-offroad-ev-concept/) 
- [article 2](https://www.engadget.com/2018/04/14/the-morning-after-weekend-edition/)
- [article 3](https://www.engadget.com/2017/11/03/xbox-one-x-review/)

![selection_18-04-19-14 52 37](https://user-images.githubusercontent.com/6399306/38992564-6406ebae-43e1-11e8-8c70-8548005649e1.png)

Thanks

**Comment by andreskrey** [comment]
Hi @gijsk 
Doesn't seems to be related to #431 because that paraghraph is not inside an `<aside>` tag. It kinda reminds me of the issue we had in #407 (single `p` inside a `div`)

I won't be able to debug it this weekend but definitely going to work on it next week. :)

**Comment by andreskrey** [comment]
You're right @gijsk, the issue comes from the fact that the first paragraph of the article ("Buick, a brand not known...") is in it's own div and the later paragraphs are nested inside another div.

The algorithm selects the "Since GM is just playing anyway..." node as the top candidate and scans everything below it. The parent node that has both divs also hosts other stuff like the comment section  and the related articles so its score is really low.

 Right now I'm kinda lost on how to fix it but I'll keep looking for a solution.
